ppa/editorial/models.py

```diff
@@ -83,15 +81,34 @@ def route(self, request, path_components):
             return super().route(request, path_components)
 
 
+@register_snippet
+class Person(models.Model):
+    name = models.CharField(max_length=255)
+    url = models.URLField(blank=True, default='')
```

Comment:
Let's add help text please. Maybe something along these lines?
- full name for the person as it should appear in the author list
- Personal website, profile page, or social media profile page for this person.

(The `URL` should be something we can use as a URI for the person when we're able to generate schema.org metadata for blogposts; there may be a better way to articulate that here in the help text.)